public/locations-list-map.js

```diff
@@ -1,5 +1,49 @@
 import toDataByLocation from './toDataByLocation.js';
 
+
+
+// TODO (patricknelson): The indentation in this file is not consistent and should be updated soon. Currently the project
+//  standard is 2-space indentation; please see .editorconfig and, if possible, configure your IDE to utilize this file.
+
+// TODO (patricknelson): All functions interacting with the map should not require the map as a parameter since this
+//  will not change in the near future and is a module level variable already. Causes too much repeated code.
+
+// TODO (patricknelson: Proposal to change functions responsible for recentering the map, using either generic lat/lng
```

Comment:
File issue instead of a TODO please.

Reply:
Same q here, hold up merge just for that?